mod-inn-reach/src/main/resources/volaris/mod-inn-reach/features/marc-transformation-options-settings.feature

```diff
@@ -14,23 +14,96 @@ Feature: MARC transformation options settings
 
     * configure headers = headersUser
 
-  @Undefined
-  Scenario: Get all MARC transformation options settings
-    * print 'Get all MARC transformation options settings'
+    * callonce variables
 
-  @Undefined
-  Scenario: Get MARC transformation options settings by id
-    * print 'Get MARC transformation options settings by id'
+    * print 'Prepare central servers'
+    * callonce read(featuresPath + 'central-server.feature@create')
+    * def centralServer1 = response.centralServers[0]
+    * def centralServer2 = response.centralServers[1]
+    * def notExistedCentralServerId1 = globalCentralServerId1
 
-  @Undefined
+  @create
   Scenario: Create MARC transformation options settings
     * print 'Create MARC transformation options settings'
+    Given path centralServer1.id + '/marc-transformation-options'
+    And request read(samplesPath + 'marc-transformation-options/create-marc-transformation-options.json')
+    When method POST
+    Then status 201
 
-  @Undefined
+    * def ResponseMARCTransformation = $
+    And match ResponseMARCTransformation.configIsActive == true
+    And match ResponseMARCTransformation.modifiedFieldsForContributedRecords[0].resourceIdentifierTypeId == "d8bd646c-5371-4167-9f66-cd6922f93eb8"
+
+  Scenario: Attempting to create MARC transformation options settings that already exist
+    * print 'Attempting to create MARC transformation options for negative scenario'
+    * def marc = read(samplesPath + 'marc-transformation-options/create-marc-transformation-options.json')
+    Given path centralServer1.id + '/marc-transformation-options'
+    And request marc
+    When method POST
+    Then status 500
+
+  @get
+  Scenario: Get MARC transformation options settings by central server id
+    * print 'Get MARC transformation options settings by central server id'
+    Given path centralServer1.id + '/marc-transformation-options'
```

Comment:
define the path in background and reuse it

Reply:
Done